docs_quicksteps.md

```diff
@@ -0,0 +1,23 @@
+# AWS CLI code example contribution quick steps
+
+```{danger}
+The AWS CLI repository is a **public** GitHub repository and submitting examples before a feature launch produces issues. You must be able to run the command from the public installation source.
+```
+
+Thank you for your interest in contributing code examples to the AWS Command Line Interface (AWS CLI)! This page provides the basic steps for making a contribution to AWS CLI code examples.
+
+## Quick steps
+
+1. [Create or update your environment](getting_started.md).
+1. Create a new branch in Git. For example Git commits and workflows, see [Git commits and workflows](git_workflows.md).
```

Comment:
We should consider explicitly calling out the feature branch workflow example on what to do here to make sure it is clear they should make a feature branch that they will push to their own fork.